java/src/main/java/org/upgradeplatform/client/ExperimentClient.java

```diff
@@ -269,4 +279,158 @@ public void failed(Throwable throwable) {
 		}));
 
 	}
+
+	public void getAllFeatureFlags(final ResponseCallback<List<FeatureFlag>> callbacks) {
+
+		AsyncInvoker invocation = this.apiService.prepareRequest(GET_ALL_FEATURE_FLAGS);
+
+		invocation.get(new PublishingRetryCallback<>(invocation, Entity.json(""), MAX_RETRIES, REQUEST_TYPES_GET,
```

Comment:
`PublishingRetryCallback` is for retry request in case it failed because of temporary network issue or something.  Yeah, i can avoid passing empty entity by creating another constructor in `PublishingRetryCallback `

Reply:
#e7a5725

Reply:
Please look at the latest commit.

Reply:
looks good, thanks!